packages/opencensus-propagation-jaeger/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "pretty": true,
+    "module": "commonjs",
+    "target": "es6",
+    "strictNullChecks": false
```

Comment:
Not sure if this `tsconfig.json` was cloned from somewhere else, but either way, would the code still compile if this option was removed?